### PR TITLE
Adjust rendering of widgets: ensure correct rendering

### PR DESCRIPTION
**Pull request by simialbi**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #1, #70 

I think it's necessary to check all of the form parts at least once again (Do all the layouts work properly and all the fields in all the layouts according to bs4 doc). I'll do that later and I'll define some more tests to check everything of this (WIP).

Additionaly I added a german translation of the guide.

### CODE REVIEW

**Comment by simialbi** [comment]
I have added the form and field tests in the meantime as mentioned before. Have a look.
I my opinion it now fixes #1 completely. Have a look

File: docs/guide-de-CH/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Twitter Bootstrap Erweiterung für Yii 2
```
Comment (samdark) [inline docs/guide-de-CH/README.md]: I think it should be `guide-de`. German in Switzerland should be OK for other German-speaking countries, right? If so, it's better to join efforts.

File: docs/guide-de-CH/basic-usage.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Basic Usage
```
Comment (samdark) [inline docs/guide-de-CH/basic-usage.md]: Translation missing.

File: docs/guide-es/usage-widgets.md [hunk 3]
```diff
@@ -5,19 +5,23 @@ La mayoría de los componentes de bootstrap están encapsulados dentro de Yii wi
...
+- [[yii\bootstrap4\ActiveField|ActiveField]]
 - [[yii\bootstrap4\ActiveForm|ActiveForm]]
 - [[yii\bootstrap4\Alert|Alert]]
 - [[yii\bootstrap4\Button|Button]]
 - [[yii\bootstrap4\ButtonDropdown|ButtonDropdown]]
 - [[yii\bootstrap4\ButtonGroup|ButtonGroup]]
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
Comment (samdark) [inline docs/guide-es/usage-widgets.md]:
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *obsoleto* (see [[yii\bootstrap4\Accordion|Accordion]])
```

File: docs/guide-ja/usage-widgets.md [hunk 4]
```diff
@@ -4,19 +4,23 @@ Yii ウィジェット
...
+- [[yii\bootstrap4\ActiveField|ActiveField]]
 - [[yii\bootstrap4\ActiveForm|ActiveForm]]
 - [[yii\bootstrap4\Alert|Alert]]
 - [[yii\bootstrap4\Button|Button]]
 - [[yii\bootstrap4\ButtonDropdown|ButtonDropdown]]
 - [[yii\bootstrap4\ButtonGroup|ButtonGroup]]
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
Comment (samdark) [inline docs/guide-ja/usage-widgets.md]:
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *廃止予定* (see [[yii\bootstrap4\Accordion|Accordion]])
```

File: docs/guide-pt-BR/usage-widgets.md [hunk 5]
```diff
@@ -4,16 +4,20 @@ Yii widgets
...
+- [[yii\bootstrap4\ActiveField|ActiveField]]
 - [[yii\bootstrap4\ActiveForm|ActiveForm]]
 - [[yii\bootstrap4\Alert|Alert]]
 - [[yii\bootstrap4\Button|Button]]
 - [[yii\bootstrap4\ButtonDropdown|ButtonDropdown]]
 - [[yii\bootstrap4\ButtonGroup|ButtonGroup]]
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
Comment (samdark) [inline docs/guide-pt-BR/usage-widgets.md]:
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *descontinuada* (see [[yii\bootstrap4\Accordion|Accordion]])
```

File: docs/guide-uk/usage-widgets.md [hunk 6]
```diff
@@ -4,16 +4,20 @@
...
+- [[yii\bootstrap4\ActiveField|ActiveField]]
 - [[yii\bootstrap4\ActiveForm|ActiveForm]]
 - [[yii\bootstrap4\Alert|Alert]]
 - [[yii\bootstrap4\Button|Button]]
 - [[yii\bootstrap4\ButtonDropdown|ButtonDropdown]]
 - [[yii\bootstrap4\ButtonGroup|ButtonGroup]]
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
Comment (samdark) [inline docs/guide-uk/usage-widgets.md]:
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *засуджується* (see [[yii\bootstrap4\Accordion|Accordion]])
```

Comment (samdark) [reply docs/guide-uk/usage-widgets.md]: @SilverFire, @hiqsol, correct?

File: docs/guide-zh-CN/usage-widgets.md [hunk 7]
```diff
@@ -3,20 +3,23 @@ Yii 小部件
...
+- [[yii\bootstrap4\ActiveField|ActiveField]]
 - [[yii\bootstrap4\ActiveForm|ActiveForm]]
 - [[yii\bootstrap4\Alert|Alert]]
 - [[yii\bootstrap4\Button|Button]]
 - [[yii\bootstrap4\ButtonDropdown|ButtonDropdown]]
 - [[yii\bootstrap4\ButtonGroup|ButtonGroup]]
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
Comment (samdark) [inline docs/guide-zh-CN/usage-widgets.md]:
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *弃用* (see [[yii\bootstrap4\Accordion|Accordion]])
```

Comment (samdark) [reply docs/guide-zh-CN/usage-widgets.md]: @kids-return correct?

File: docs/guide-de-CH/README.md (hunk 1 above)
Comment (simialbi) [reply docs/guide-de-CH/README.md]: Of course it's nearby the same. There is just one (little) difference the germans have to the swiss (and that's why I named it de-CH): [the ß char](https://en.wikipedia.org/wiki/%C3%9F)
But that's no reason they wont understand 😄 . I'll refactor it.

**Comment by simialbi** [comment]
I changed the translations of `deprecated` like you suggested and copied the Breadcrumbs-Widget from #70  and added a rendering test. I hope we can solve both pr's with these changes.

**Comment by samdark** [comment]
Merged. Thank you very much!

**Pull request merged**